templates/content/field-group-html-element--node--group-action-bar.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+/**
+ * @file
+ * Theme override for the "Action bar" fieldgroup html element.
+ *
+ * Available variables:
```

Comment:
these are already present in the parent template no use to keep track and maintain these, same in the other templates

Reply:
Removed the variables docs